app/javascript/packs/fundraiser.js

```diff
@@ -88,3 +88,5 @@ window.mountFundraiser = function(root: string, data: MountFundraiserOptions) {
     });
   }
 };
+
+window.storeDispatch = dispatch;
```

Comment:
Had you intended on keeping this global?

Reply:
yeah, I need to expose the store to our petition backbone app. Is there any other way to do this?

Reply:
Left a comment on the other PR: the redux store is in `window.champaign.store` so you'll have access to `store.dispatch()`, `store.getState()`, `store.subscribe()`, etc.

Reply:
good idea, I'll change it

Reply:
Actually, the store was already in `window.champaign.store`. Changing the code to use that.